nerfstudio/models/splatfacto.py

```diff
@@ -80,6 +80,21 @@ def SH2RGB(sh):
     return sh * C0 + 0.5
 
 
+def resize_image(image: torch.Tensor, d: int):
+    """
+    Downscale images using the same 'area' method in opencv
```

Comment:
Hi @jb-ye, just wondering do you have any reference for this idea? Would like to learn more, thanks.

Reply:
I found that the absgrad is rather sensitive to how we downscale images during coarse-to-fine training. The previous implementation is not antialiased, and would introduce more noises in computing absgrad. The new downscale method in this PR is basically the same [INTER_AREA](https://medium.com/@wenrudong/what-is-opencvs-inter-area-actually-doing-282a626a09b3) method in opencv. This approach is actually recommended for most NeRF based approach for downscaling images.

Reply:
actually OpenCV's `INTER_AREA` is a bit different because it also works with non-integer scales.

I would say this change is closer to [skimage.transform.downscale_local_mean](https://scikit-image.org/docs/stable/api/skimage.transform.html#skimage.transform.downscale_local_mean)